panther_analysis_tool/log_schemas/user_defined.py

```diff
@@ -38,13 +40,14 @@ class Client:
     _LIST_SCHEMAS_ENDPOINT = "ListSchemas"
     _PUT_SCHEMA_ENDPOINT = "PutUserSchema"
 
-    def __init__(self) -> None:
+    def __init__(self, args: argparse.Namespace) -> None:
```

Comment:
nit: pass an `aws_profile=None` argument here and assign a value in `main`.

Reply:
Either that or maybe use a slightly different name such as `options`? As you may notice the attribute name `_args` does not provide enough context that it refers to command-line arguments.